README.md

```diff
@@ -1,5 +1,70 @@
 # Learning analytics playground
 
+## Description
+This tool is a first draft of connecting an LMS interface to a Graylog server to create a dataset 
+of learning logs.
+
+It is intended to be used locally. 
+Currently, it is using LMS from OpenEdx and the FUN's release of dogwood.
+
+## Starting with the tool
+
+1. Clone the current repository.
+
+2. To activate settings for the `dogwood` release used in the project, run the following command:
+
+    ```bin/activate```
```

Comment:
Code can be written without indentation but using triple back quotes notation :

```suggestion
'''
bin/activate
'''
```

__nota bene__: the previous suggestion does not have back quotes but quotes, 'cause I cannot escape them :stuck_out_tongue: